aodh/files/heka.toml

```diff
@@ -0,0 +1,13 @@
+[logstreamer_ceilometer]
+type = "LogstreamerInput"
+log_directory = "/var/log/aodh"
+file_match = '(?P<Service>.+)\.log\.?(?P<Index>\d*)?(.gz)?'
+differentiator = ["'aodh','_','Service'"]
+priority = ["^Index"]
+decoder = "openstack"
+oldest_duration = "168h"
+
+[openstack]
+type = "SandboxDecoder"
+filename = "lua_modules/decoders/openstack.lua"
+module_directory = "/usr/share/heka/lua_modules;/usr/share/heka/lua_modules/common"
```

Comment:
You should not have this file. All the Heka configuration related to Aodh should be in `aodh/meta/heka.yml`.